deltacat/tests/local_deltacat_storage/__init__.py

```diff
@@ -188,9 +188,9 @@ def list_deltas(
 
     for delta in all_deltas:
         if (
-            not first_stream_position or first_stream_position <= delta.stream_position
```

Comment:
Do we change here from [first_stream_position, last_stream_position) to (first_stream_position, last_stream_position]? What's the context of this change?

Reply:
It was bug.